### ISSUE TITLE
Form Input

### ISSUE THREAD

**Issue by CruzST** [domdom258/CS467_project#19]
SpecificProduct.vue

Form Input for the number to add to cart. For some reason it's not recognizing min and max attributes.

Code that youd expect to work. but it doesnt.
`<b-form-input v-model="qty" type="number" min="1" :max="productObject.Quantity"></b-form-input>`

**Comment by michelle-tan** [comment]
I believe html input validation works when you put the input within a form, but not with a standalone input and separate button. Do we want html validation? We can also write our own validation to disable the button when the qty exceeds the productObject.qty, for example.

**Comment by CruzST** [comment]
Ah I think its a browser issue and it's reverting to a text type.  Validation would be good, since we can technically add 0 or more than the quantity to the cart. I think you're right, disabling the button if an invalid number is input would be best

**Comment by CruzST** [comment]
![image](https://user-images.githubusercontent.com/30658915/59161467-f58c4a80-8ab0-11e9-9ce9-9e4e8daa0ca5.png)

![image](https://user-images.githubusercontent.com/30658915/59161474-063cc080-8ab1-11e9-8ddb-f4caa8cd1a8a.png)

![image](https://user-images.githubusercontent.com/30658915/59161477-0f2d9200-8ab1-11e9-99af-469b3e3365fb.png)

![image](https://user-images.githubusercontent.com/30658915/59161479-194f9080-8ab1-11e9-9055-6a3eee2aa898.png)


added validation manually.  Added a invalid feedback message too. I had to create a div for itbecause boostrap was not being compliant.  The validation disables the button if it is 0 or greater than the items stock
